generate_tags.py

```diff
@@ -0,0 +1,83 @@
+import json
+
+import requests
+
+# This script loads the enterprise-attack.json mitre dataset into a dict (mitre_dataset)
+# according to the parameters set below. The dict is then iterated to produce an attack.py
+# file, containing variables for the multiple Mitre ATT&CK tags relevant to the project.
+#
+# The enterprise-attack.json file is sourced from Mitre's repository at
+# https://github.com/mitre-attack/attack-stix-data/tree/master/enterprise-attack.
+
+# Replace the URL below with the version of your choice.
+
+mitre_source = (
+    "https://raw.githubusercontent.com/mitre-attack/attack-stix-data/"
+    "master/enterprise-attack/enterprise-attack-14.1.json"
+)
+
+# Begin parameters; modify these to customize the tags produced by the script
+
+included_platforms = ["Windows", "Linux"]
+
+# End parameters
+
+
+def retrieve_mitre_data(url):
+    print("Attempting download of Mitre data, please wait...")
+    response = requests.get(url)
+    if response.status_code == 200:
+        print("\nDownload finished!")
+    else:
+        print("Error: ", response.status_code)
+    return json.loads(response.content)
```

Comment:
See https://3.python-requests.org/api/?highlight=raise_for_status#requests.Response.raise_for_status
```suggestion
    response.raise_for_status()
    
    print("\nDownload finished!")
    return json.loads(response.content)
```

If you do this, you don't need `if mitre_dataset` on line 61.